recipes-bsp/rpi-u-boot-scr/rpi-u-boot-scr.bb

```diff
@@ -13,10 +13,10 @@ do_compile() {
     sed -e 's/@@KERNEL_IMAGETYPE@@/${KERNEL_IMAGETYPE}/' \
         -e 's/@@KERNEL_BOOTCMD@@/${KERNEL_BOOTCMD}/' \
         "${WORKDIR}/boot.cmd.in" > "${WORKDIR}/boot.cmd"
-    mkimage -A arm -T script -C none -n "Boot script" -d "${WORKDIR}/boot.cmd" boot.scr
+    mkimage -A ${UBOOT_ARCH} -T script -C none -n "Boot script" -d "${WORKDIR}/boot.cmd" boot.scr
```

Comment:
What is mkimage doing with the arch setting?

Reply:
lets accept it, maybe tomorrow RPI foundation will have a rpi which is not based on arm who knows :)

Reply:
I assumed a test was done, @liuming50 can you confirm?

Reply:
Yes, I have tested it on a raspberrypi4-64 and raspberrypi3 device.

Reply:
Thanks for confirming. Merged.